src/Theming/Theme.types.ts

```diff
@@ -0,0 +1,24 @@
+import { IPalette, IPartialTheme, Theme } from '@fluentui/react';
+
+/** custom colors our app needs */
+export interface CustomPalette {
```

Comment:
should this interface extend from a base pallette or something? otherwise, how can we make sure that all the <key, value> pairs are valid palette variables? or should we add [key:string]: string to the interface when we need to extend the varibles like glassyBorder? I think my main question is about how we decide the names like glassyBackground75, are they coming from a base interface?